### PR TITLE
Rename discord-gated-entry to discord-bot

### PR DESCRIPTION
**Pull request by raccube**
### Testing

- [x] applied the configuration locally
- [ ] manually validated the new behaviour

### CODE REVIEW

File: roles/discord-bot/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Discord Gated Entry
+
+A Bot for gating entry to a Discord server.
```
Comment (WillB97) [inline roles/discord-bot/README.md:3]: You missed one.

Comment (raccube) [reply roles/discord-bot/README.md:3]: Good catch

Comment (raccube) [reply roles/discord-bot/README.md]: b070a26

File: roles/discord-bot/tasks/main.yml [hunk 2]
```diff
@@ -20,12 +20,12 @@
 
 - name: Download
   git:
-    repo: https://github.com/srobo/discord-gated-entry
+    repo: https://github.com/srobo/discord-bot
     dest: "{{ install_dir }}"
     force: true
     version: 476a9c942f580e2950048c2b95e15429cb0ae33d
```
Comment (RealOrangeOne) [inline roles/discord-bot/tasks/main.yml:26]: Question: Should the commit be updated at the same time?

File: roles/discord-bot/tasks/main.yml [hunk 3]
```diff
@@ -20,12 +20,12 @@
...
   notify:
-    Restart discord-gated-entry
+    Restart discord-bot
   register: discord_gated_entry_repo
```
Comment (RealOrangeOne) [inline roles/discord-bot/tasks/main.yml:29]: Nit: this variable should be updated

File: roles/discord-bot/templates/discord-gated-entry.service [hunk 4]
```diff
@@ -8,9 +8,9 @@ User=discord
 Type=simple
 
 WorkingDirectory={{ install_dir }}
-RuntimeDirectory=discord-gated-entry
+RuntimeDirectory=discord-bot
 
-ExecStart=/srv/discord-gated-entry/venv/bin/python main.py
+ExecStart=/srv/discord-bot/venv/bin/python src/main.py
```
Comment (RealOrangeOne) [inline roles/discord-bot/templates/discord-gated-entry.service:13]: Suggestion: Might as well make things better whilst we're here

```suggestion
ExecStart={{ venv_dir }}/bin/python src/main.py
```

**Review by RealOrangeOne — APPROVED** [review]
I've not tested, but in principle these look good.

I suggest we pretend the old discord bot doesn't exist, and manually clean that up before deploying this.

File: roles/discord-bot/tasks/main.yml (hunk 2 above)
Comment (raccube) [reply roles/discord-bot/tasks/main.yml:26]: Yes, it should.

**Review by RealOrangeOne — APPROVED** [review]
Looks good to me, and tested it executes in vagrant.

**Pull request merged**